protocols/upnp/src/provider.rs

```diff
@@ -35,15 +94,17 @@ pub struct Gateway {
     pub(crate) external_addr: IpAddr,
 }
 
+impl Gateway {}
```

Comment:
What is this for? :)

Reply:
aha nothing, was left dangling. Thanks Thomas!